src/components/ColumnList.tsx

```diff
@@ -0,0 +1,25 @@
+import { Card, Column, WithID } from '../store/store'
+import { ColumnItem } from './ColumnItem'
+
+interface Props {
+  cards: WithID<Card>[]
+  columns: WithID<Column>[]
+}
+
+export const ColumnList = ({ cards, columns }: Props) => {
+  return (
+    <>
+      {columns.map(
+        (column) =>
+          column && (
```

Comment:
You should not need that check. At least from type's perspective. Try to remove it. We can think in the future about implementing decoders for checking data we get from the backend.
```suggestion
          (
```